godot-core/src/registry/class.rs

```diff
@@ -477,6 +479,11 @@ fn fill_class_info(item: PluginItem, c: &mut ClassRegistrationInfo) {
             c.godot_params.property_can_revert_func = user_property_can_revert_fn;
             c.godot_params.property_get_revert_func = user_property_get_revert_fn;
             c.user_virtual_fn = get_virtual_fn;
+            // Not present in GdExtension.
```

Comment:
```suggestion
            // Not present in GDExtension.
```
Also, what do you mean by that? `validate_property_func` is clearly available...

Reply:
I meant that it wasn't present in GDExtension before 4.2 :sweat_smile: – I decided to remove this comment altogether, since it is just a noise (`#[cfg(since_api = "4.2")]` tells all one needs to know)